docs/faq.md

```diff
@@ -81,3 +81,14 @@ sudo systemctl disable cloud4rpi.service
 sudo rm /lib/systemd/system/cloud4rpi.service
 sudo systemctl daemon-reload
 ```
+
+
+## How to choose the DATA_SENDING_INTERVAL value?
+
+You should call [device.publish_data()](/api/python/#publish_data) and [device.publish_diag()](/api/python/#publish_diag) functions frequently to update the monitored variables. The recommended update frequency depends on the variable dynamics, for example, atmospheric conditions
+ do not change every minute. This means the delay between updates for a weather station should be one to ten minutes.
+
+If you monitor several variables with different dynamics, the delay between updates should be the shortest of all.
+
+!!! Note
+    The variable update interval does not affect commands reception. Variable changes made using the Web UI (**Switch** or **Slider** widget) are applied instantly.
```

Comment:
commands reception?

Reply:
The Cloud sends commands to devices. Devices receives these commands. Instantly.

Reply:
The variable update interval does not affect commands, and variable changes made through the Web UI (**Switch** or **Slider** widget) are applied instantly.